app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/categories.controllers");
+
+app.use(express.json());
```

Comment:
Do you know what this does? It parses the body that is sent through, so therefore it's not needed until you start to send data (i.e. a POST or PATCH request). Make sure you understand the code that is going into your repo.